pkg/driverbuilder/builder/image_test.go

```diff
@@ -233,6 +253,37 @@ func TestImagesListingFromFile(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		assert.DeepEqual(t, expected, lister.LoadImages())
+		assert.DeepEqual(t, test.expected, lister.LoadImages())
+	}
+}
+
+func TestRepoImagesLister(t *testing.T) {
```

Comment:
Both FileImagesLister and RepoImagesLister use same tests, so that we can be sure they are coherent with each other.